config/locales/devise.en.yml

```diff
@@ -35,7 +35,7 @@ en:
       updated_not_active: "Your password has been changed successfully."
     registrations:
       destroyed: "Bye! Your account has been successfully cancelled. We hope to see you again soon."
-      signed_up: "Welcome to DC Health Link. Your account has been created."
+      signed_up: "Welcome to Health Connector. Your account has been created."
```

Comment:
check if this can reference Settings directly --- may not be possible because of load order